integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/di/CoinBaseModule.kt

```diff
@@ -62,6 +63,8 @@ object CoinBaseModule {
     fun providePlaceBuyOrderMapper(): PlaceBuyOrderMapper = PlaceBuyOrderMapper()
     @Provides
     fun provideCommitBuyOrderMapper(): CommitBuyOrderMapper = CommitBuyOrderMapper()
+    @Provides
+    fun provideSwapTradeMapper(): SwapTradeMapper = SwapTradeMapper()
```

Comment:
`@Provides` is for providing an existing instance of a class. If a new instance is created, you can use `@Binds` - see `ExploreDashModule`.